libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapRouteLine.kt

```diff
@@ -365,6 +365,7 @@ internal class MapRouteLine(
             )
             routeFeatureData.addAll(newRouteFeatureData)
             drawRoutes(newRouteFeatureData)
+            hideShieldLineAtOffset(0f)
```

Comment:
Curious, why do we hide the shield when drawing the routes? :thinking:

Reply:
I noticed that when drawing a new route after ending navigation the shield layer had a piece missing. In order to get the line to look correct I redraw the shield layer.

Reply:
Makes sense. Although it seems you  meant "remove" the shield layer (instead of "redraw") i.e. redraw an empty shield layer, right?

Reply:
@Guardiola31337 I should be clearer. It's hiding the shieldLine at a starting point of 0. When the shield and route line gradients are created the color changes are based on the percentage of the line. So hideShieldLineAtOffset(0f) means the transparent part starts at 0% and the colored part of the line is in effect 100% which shows the whole line.

Reply:
I think we're saying the same. Thanks for clarifying 👍 We should be good!